Source/Widgets/InfoBox/InfoBox.js

```diff
@@ -155,6 +171,32 @@ click: function () { closeClicked.raiseEvent(this); }');
         },
 
         /**
+         * Gets or sets the security settings for iFrames generated by the Infobox.
+         * This will disable sandboxing for the iFrame, and allow HTML to be in the title.
+         * Only do this if you are in complete control of the data that will be loaded into the InfoBox,
+         * otherwise it is a security issue.
+         * @memberof InfoBox.prototype
+         *
+         * @type {Boolean}
+         */
+        /*disableSecurity : {
+            get : function() {
+                return this._disableSecurity;
+            },
+            set : function(value) {
+                this._disableSecurity = value;
+                this._viewModel.securityDisabled = value;
+
+                if (value) {
+                    this._frame.removeAttribute('sandbox');
+                }
+                else {
+                    this._frame.setAttribute('sandbox', 'allow-same-origin allow-popups allow-forms');
+                }
+            }
+        },*/
```

Comment:
Remove this commented-out code, and move the more detailed documentation to the new property in the viewModel please.